.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/dss-test"]
 	path = lib/dss-test
 	url = https://github.com/makerdao/dss-test
+[submodule "lib/solmate"]
```

Comment:
I think it would be better to not include a third party git submodule and just copy the needed files for testing in a tests/utils or tests/helpers directory.

Reply:
ok, will remove

Reply:
but I think then I think that we need to keep the soulmate licence.
otherwise I can copy-paste the oz 4626

Reply:
Ok good, let's leave this change on hold until getting closer to merge the PR.